clients/feeder/event_listener.go

```diff
@@ -0,0 +1,17 @@
+package feeder
+
+import "time"
+
+type EventListener interface {
+	OnStatus200Response(urlPath string, took time.Duration)
+}
+
+type SelectiveListener struct {
+	OnStatus200ResponseCb func(urlPath string, took time.Duration)
+}
+
+func (l SelectiveListener) OnStatus200Response(urlPath string, took time.Duration) {
```

Comment:
How about something like;

```suggestion
func (l SelectiveListener) OnResponse(urlPath string, status http.StatusCode, took time.Duration) {
```